en-US/features/custome_template.md

```diff
@@ -14,6 +14,6 @@ Here is an example of how you can include a custom CSS file to your Gogs instanc
 
 1. Create a file named `custom.css` under `public/css` directory.
 2. Put some CSS rules into the file.
-3. Edit file `custom/templates/inject/head.tmpl` and add a line `<link rel="stylesheet" href="/css/custom.css">`
```

Comment:
Hi... you cannot delete prefix `custom/`, otherwise you're touching the main repository Git history which is not what you want.